server/api.js

```diff
@@ -91,4 +91,40 @@ router.get("/laptop_donation", async (req, res) => {
 	}
 });
 
+// post laptop assignment
+router.post("/laptop_assignment", (req, res) => {
+	let laptopRequestsId = req.body.laptop_request_id;
+	let laptopDonationId = req.body.laptop_donation_id;
+	let status = req.body.status;
+	let assignmentId = req.params.laptop_assignment;
```

Comment:
there is no laptop_assignment in the params -> this is part of the point of a POST request: we use it when we don't have an id yet

Reply:
Done